modules/node-network-configuration-policy-file.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-observing-node-network-state.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-network-configuration-policy-file.adoc_{context}"]
+= The NodeNetworkConfigurationPolicy manifest file
+
+A `NodeNetworkConfigurationPolicy` (NNCP) manifest file defines policies that the Kubernetes NMState Operator uses to configure networking for nodes that exist in an {product-title} cluster. 
+
+After you apply a node network policy to a node, the Kubernetes NMState Operator creates an interface on the node. A node network policy includes your requested network configuration and the status of execution for the policy on the cluster as a whole.
+
+You can create an NNCP by using either the {oc-first} or the {product-title} web console. As a postinstallation task you can create an NNCP or edit an existing NNCP.
+
+[NOTE]
+====
+Before you create an NNCP, ensure that you read the "Example policy configurations for different interfaces" document.
+====
+
+If you want to delete an NNCP, you can use the `oc delete nncp` command to complete this action. However, this command does not delete any created objects, such as a bridge interface. 
+
+Deleting the node network policy that added an interface to a node does not change the configuration of the policy on the node. Similarly, removing an interface does not delete the policy, because the Kubernetes NMState Operator recreates the removed interface whenever a pod or a node is restarted.
+
+To effectively delete the NNCP, the node network policy, and any created interfaces would typically require the following actions:
+
+. Edit the NNCP and remove all interface details from the file.
```

Comment:
I think this can be confusing. While as a Nmstate developer I know that the interface details usually refers to non-identifier values like mtu, ip configuration and so, a user that is not very familiar could think it refers to all under the `interfaces` section.

It could be better to explicitly say that `name`, `state` and `type` shouldn't be removed.

Reply:
Thanks for the details. I updated the doc to state what should not be removed.